infrastructure/lib/stacks/opensearchNginxProxyReadonly.ts

```diff
@@ -65,8 +65,8 @@ export class OpenSearchMetricsNginxReadonly extends Stack {
 
         const instanceRole = this.createNginxReadonlyInstanceRole(props);
             this.asg = new AutoScalingGroup(this, 'OpenSearchMetricsReadonly-MetricsProxyAsg', {
-            instanceType: InstanceType.of(InstanceClass.M5, InstanceSize.LARGE),
-            blockDevices: [{ deviceName: '/dev/xvda', volume: BlockDeviceVolume.ebs(10) }], // GB
+            instanceType: InstanceType.of(InstanceClass.M5, InstanceSize.XLARGE),
+            blockDevices: [{ deviceName: '/dev/xvda', volume: BlockDeviceVolume.ebs(50) }], // GB
```

Comment:
Is the nginx under pressure and cpu is under pressure?
Also, are you persisting any data, like logs on the host that is causing the disk full problems?

Reply:
The docker logs for now are persisted until we add the ` --log-driver awslogs` and push to cloudwatch, so the overlay is getting filled up quickly with lot of access logs and requests, hence updated to 50 (along with `-log-opt max-size=50m --log-opt max-file=5 `), also the `LARGE` is just having 2 CPU. Untill we have the proper docker-compose file and add proper resourcing we might need to go with `XLARGE`. Thanks